connect-example/src/main/java/com/stripe/android/connect/example/ui/appearance/AppearanceViewModel.kt

```diff
@@ -0,0 +1,72 @@
+package com.stripe.android.connect.example.ui.appearance
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.stripe.android.connect.BuildConfig
+import com.stripe.android.connect.example.data.SettingsService
+import com.stripe.android.core.Logger
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
+class AppearanceViewModel(
+    private val logger: Logger = Logger.getInstance(enableLogging = BuildConfig.DEBUG),
+    private val settingsService: SettingsService = SettingsService.getInstance(),
+) : ViewModel() {
+
+    private val _state = MutableStateFlow(AppearanceState())
+    val state = _state.asStateFlow()
+
+    init {
+        loadAppearance()
+
+        viewModelScope.launch {
+            _state.map {
+                // when any of the below values change,
+                // enable the save button
+                listOf(it.selectedAppearance)
```

Comment:
can you just do `it.selectedAppearance`? Do you need the listOf?

Reply:
Good point. The listOf was a copy from the other ViewModel, your suggestion is better.